views/includes/check-your-answers/beneficial-owner-gov.html

```diff
@@ -98,7 +98,7 @@ <h3 class="govuk-heading-m">Government or public authority beneficial owner</h3>
             changeLinkUrl + "/" + bog.id + OE_CONFIGS.NAME,
             "Government or public authority beneficial owner " + bog.name + " - name",
             "change-government-or-public-authority-beneficial-owner-name-button"
-          ) ]
+          ) if pageParams.isRegistration ]
```

Comment:
Needs testing of registration but I'm surprised this is registration only?

Reply:
if  not pageParams.isStatement 
?

Reply:
Fixed